Cargo.toml

```diff
@@ -65,3 +65,8 @@ failure_derive = { version = "0.1.5", optional = true}
 [dev-dependencies]
 pretty_assertions = "0.6.1"
 tempfile = "^3.1.0"
+criterion = "0.2"
+
+[[bench]]
+name = "osrank_naive"
+harness = false
```

Comment:
Out of curiosity (too lazy to read the docs 😬 ). What does the `harness` option do?

Reply:
I have to admit I just copied it from the [getting started guide] 😬(https://bheisler.github.io/criterion.rs/book/getting_started.html#step-1---add-dependency-to-cargotoml)